oio/container/client.py

```diff
@@ -310,7 +310,7 @@ def content_create(self, account=None, reference=None, path=None,
         if chunk_method is not None:
             hdrs['x-oio-content-meta-chunk-method'] = chunk_method
         resp, body = self._direct_request(
-            'POST', uri, data=data, params=params, autocreate=True,
+            'POST', uri, data=data, params=params,
```

Comment:
You should add `append` parameter to the method signature, and then set it in `params`.